extensions/notebook/resources/notebooks/JupyterBooksCreate.ipynb

```diff
@@ -26,6 +26,9 @@
                 "# Create Jupyter Book\n",
                 " <span style=\"color:red\">**Note**:</span> **Windows users** using **Python 3.8** might be impacted by this known [issue](https://jupyterbook.org/advanced/advanced.html#working-on-windows) on the latest **jupyter book** version.\n",
                 "\n",
+                "  <span style=\"color:red\">**Note**:</span> In versions above 0.7.0, verify that the names of the files do not start with a **number**. Due to the new design of jupyter book, the prefix number will be removed from the file name in the Table of Contents. Please refer to this [section](https://jupyterbook.org/customize/toc.html#automatically-generate-your-toc-yml-file) of the documentation for more information.\n",
```

Comment:
we should say something about "Jupyter Book"

"In Jupyter Book versions above 0.7.0" would be clearer